web/src/DetailedInfoPage.tsx

```diff
@@ -526,7 +516,7 @@ export const DetailedInfoPage: React.FunctionComponent<{}> = () => {
             {projectName && !isLoading && !error && (
                 <>
                     <div className="detailedInfo">
-                        <div>{renderTitle()}</div>
+                        <div>{renderTitle}</div>
                         <br />
                         <div>{renderPedigreeSection()}</div>
```

Comment:
Similar treatment for all these functions because they're basically just constants.

Reply:
The renderTitle was a simple one, the others have if statements and early returns which was the whole reason I made them functions. I thought it was preferable to making them variables and either having to do `let var` and then updating them or doing a bunch of nested checks. Is it preferable for them to not be functions?